internal/configs/version1/config.go

```diff
@@ -191,6 +191,7 @@ type Location struct {
 
 // MainConfig describe the main NGINX configuration file.
 type MainConfig struct {
+	AccessLog                          string
 	AccessLogOff                       bool
```

Comment:
@hafe is this value still being used anywhere?

Reply:
Yes can be removed.

Reply:
@hafe can you please remove this var?